### PR TITLE
Replies Page

### PR DESCRIPTION
**Pull request by p0o**

### CODE REVIEW

**Comment by adcpm** [comment]
@p0o is this supposed to be similar to recent-replies page on steemit?
I get very old content on replies page please compare:
http://localhost:3000/@ekitcho/replies
https://steemit.com/@ekitcho/recent-replies

File: src/post/Feed/PostFeedCard.js [hunk 1]
```diff
@@ -72,61 +72,92 @@ const PostFeedCard = ({
...
+        <BodyShort body={post.body} />
+      </div>
+
+      {_.has(embeds, '[0].embed') &&
+        <div className="PostFeedCard__thumbs" dangerouslySetInnerHTML={{ __html: embeds[0].embed }} />
+      }
+
 
     <div className="PostFeedCard__cell PostFeedCard__cell--body">
       <h2>
```
Comment (adcpm) [inline src/post/Feed/PostFeedCard.js]: h2 is not close

Comment (p0o) [inline src/post/Feed/PostFeedCard.js]: yes I already fixed that in my local thanks

**Comment by p0o** [comment]
@adcpm should work fine now.

**Pull request merged**